pkg/printer/workload_source_printer_test.go

```diff
@@ -183,3 +183,156 @@ branch:   my-branch
 		})
 	}
 }
+
+func TestWorkloadResourcesPrinter(t *testing.T) {
+	defaultNamespace := "default"
+	workloadName := "my-workload"
+
+	tests := []struct {
+		name           string
+		testWorkload   *cartov1alpha1.Workload
+		expectedOutput string
+	}{{
+		name: "various resources",
+		testWorkload: &cartov1alpha1.Workload{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      workloadName,
+				Namespace: defaultNamespace,
+			},
+			Status: cartov1alpha1.WorkloadStatus{
+				Resources: []cartov1alpha1.RealizedResource{{
+					Name: "source-provider",
+					Conditions: []metav1.Condition{
+						{
+							Type:   cartov1alpha1.ResourceReady,
+							Status: "True",
+						},
+						{
+							Type:   cartov1alpha1.ResourceSubmitted,
+							Status: "True",
+						},
+					},
+				}, {
+					Name: "deliverable",
+					Conditions: []metav1.Condition{
+						{
+							Type:   cartov1alpha1.ResourceReady,
+							Status: "Unknown",
+						},
+						{
+							Type:   cartov1alpha1.ResourceSubmitted,
+							Status: "Unknown",
+						},
+					},
+				}, {
+					Name: "image-builder",
+					Conditions: []metav1.Condition{
+						{
+							Type:   cartov1alpha1.ResourceReady,
+							Status: "False",
+						},
+						{
+							Type:   cartov1alpha1.ResourceSubmitted,
+							Status: "False",
+						},
+					},
+				}},
+			},
+		},
+		expectedOutput: `
+RESOURCE          READY     TIME
+source-provider   True      <unknown>
```

Comment:
Issue: For ready condition `TIME` should not be `<unknown>`
```
RESOURCE          READY     TIME
source-provider   True      <unknown>
```
```suggestion
RESOURCE          READY     TIME
source-provider   True      <unknown>
```